src/applications/disability-benefits/526EZ/containers/EVSSClaimsGate.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import AlertBox from '@department-of-veterans-affairs/formation/AlertBox';
+
+import RequiredLoginView from '../../../../platform/user/authorization/components/RequiredLoginView';
+import backendServices from '../../../../platform/user/profile/constants/backendServices';
+
+export function EVSSClaimsGate({ user, location, children }) {
+  // Short-circuit the check on the intro page
+  if (location.pathname === '/introduction') {
+    return children;
+  }
+
+  if (user.login.currentlyLoggedIn && !user.profile.services.includes(backendServices.EVSS_CLAIMS)) {
```

Comment:
Does this in a way duplicate what `RequiredLoginView` already does? If the required service isn't in the user's services array, RequiredLoginView renders its children and also passes `isDataAvailable = false` to them. So, this could live in the `ITFWrapper` if we were using the standard pattern. Not sure how much difference it makes at the end of the day.

Reply:
Yeah, but here's where it gets hairy:
1) We want to render the intro page without requiring the veteran to log in yet
2) We want to handle the gating in this component instead of the child component(s)
3) Implementing _another_ component to put between `RequiredLoginView` and `children` to check for the `isDataAvailable` prop is possible, but I think would be more difficult to reason about than doing it this way

It _is_ duplicative, but it's for simplicity. :man_shrugging: